var/spack/repos/builtin/packages/py-maestrowf/package.py

```diff
@@ -42,11 +33,22 @@ class PyMaestrowf(PythonPackage):
     version("1.1.0", sha256="1bfec546831f2ef577d7823bb50dcd12622644dad0d3d761998eafd0905b6977")
     version("1.0.1", sha256="dd42ffeac1f0492a576c630b37e5d3593273e59664407f2ebf78d49322d37146")
 
+    version(
+        "1.1.9dev1",
+        tag="1.1.9dev1",
+        commit="097e6b842fd4a9540432a9dec1e41f0d1a2c2c2f",
+        deprecated=True,
+    )
+    version(
+        "1.1.7dev0",
+        sha256="bcef838f13da396dd33cc7f503655de7a8f16ee5fe7b1e2a553044334a03f1f0",
+        deprecated=True,
+    )
+
     depends_on("python@2.7:2.8,3.5:", type=("build", "run"))
     depends_on("python@3.6:", type=("build", "run"), when="@1.1.9:")
     depends_on("py-setuptools", type=("build"), when="@:1.1.8")
     depends_on("py-poetry@0.12:", type=("build"), when="@1.1.9:")
```

Comment:
According to [pyproject.toml#L68](https://github.com/LLNL/maestrowf/blob/v1.1.9/pyproject.toml#L68) this should be 
```suggestion
    depends_on("py-poetry-core@1.0.8:", type=("build"), when="@1.1.9:")
```